pisa/stages/flux/daemon_flux.py

```diff
@@ -198,7 +206,7 @@ def init_test(**param_kwargs):
     """Initialisation example"""
     param_set = []
     random_state = get_random_state(random_state=666)
-    for i, pname in enumerate(Flux(location='IceCube', use_calibration=True).params.known_parameters):
+    for i, pname in enumerate(Flux(location='IceCube', use_calibration=True, cal_file = self.cal_file).params.known_parameters):
```

Comment:
The `init_test` is not a function of the daemon_flux stage class, so it will not know the `self` reference. You can just pass `cal_file = None` for the test.